recipes/install-redis-server.rb

```diff
@@ -0,0 +1,6 @@
+# Cookbook Name:: mh-opsworks-recipes
+# Recipe:: install-redis-server
+
+::Chef::Recipe.send(:include, MhOpsworksRecipes::RecipeHelpers)
+install_package('redis-server')
```

Comment:
checked that the default is to save via both methods (aof, rdb). i think i'll leave this way. are you worried about disk?